core-bundle/composer.json

```diff
@@ -113,7 +113,7 @@
         "symfony/event-dispatcher": "^5.4 || ^6.0",
         "symfony/event-dispatcher-contracts": "^2.0 || ^3.0",
         "symfony/expression-language": "^5.4 || ^6.0",
-        "symfony/filesystem": "^5.4 || ^6.0",
+        "symfony/filesystem": "^5.4.25 || ^6.2.12",
```

Comment:
This change should be done in the `manager-bundle`, not the `core-bundle` (see https://github.com/contao/contao/pull/6171#discussion_r1241859360).

Reply:
Done in 5e9ea4773d19de886cf0d33644546ea5b479408b